project.clj

```diff
@@ -1,9 +1,9 @@
-(defproject clj-facebook-graph "0.4.0"
+(defproject yayitswei/clj-facebook-graph "0.4.2"
```

Comment:
The changes look good so far. But I first have time on the weekend to try out the code. Only seen that the fork specific name is still in there.